### PR TITLE
docs: add issue_template

### LINKED ISSUE #1593: docs: add issue template
-> Issue template provides ease for the folks who want to open an issue.
-> It differentiates the various types of issues such as `Code, Bugs, Docs, Enhancements and Others`
-> It also follows community standards.

### PR DESCRIPTION
**Pull request by bhavyastar**
Closes #1593

Created an Issue Template that has the following features Bug Report, Enhancement Feature, etc.
Would be happy to do more changes if required.
cc: @agardnerIT @mowies

### CODE REVIEW

**Comment by mowies** [comment]
I think it would be better to have a "documentation issue" template. I don't think feature request and bug really apply for this docs repo

**Comment by bhavyastar** [comment]
Hey, @mowies You are right but I think from that perspective that sometimes we want to add new content to documentation (that's why in the comment I mention if one has any suggestion related to add new content in the documentation that **comes under feature request for documentation.** ). Also, there is a little bit of improvement needed on the documentation side so for this I added a **bug request.** 
But Is it okay if I will add a `Content Request and Improvement Needed `  instead of   `Feature Request and Bug Request`  respectively? What's your thought on this?

**Comment by mowies** [comment]
I like the new titles better :) sounds like a good idea

**Comment by bhavyastar** [comment]
Hey, @mowies. Can you review this again?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: Improvement Needed
+about: Report any improvements that the documentation needs.
+labels:
+- kind/bug
```
Comment (mowies) [inline .github/ISSUE_TEMPLATE/bug-report.md:5]:
```suggestion
- bug
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Keptn Slack
+    url: https://keptn.sh/community/#slack
+    about: Please ask and answer questions here
```
Comment (mowies) [inline .github/ISSUE_TEMPLATE/config.yml:5]: [nitpick]: pls add the missing newline

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Content Request 
+about: Suggest any content recommendations for the Keptn Documentation
+title: ''
+labels: 'enhancement'
+assignees: ''
```
Comment (mowies) [inline .github/ISSUE_TEMPLATE/feature-request.md:6]: Do you need the empty fields here?
```suggestion
about: Suggest any content recommendations for the Keptn Documentation
labels: 'enhancement'
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+**This is a Feature Request**
```
Comment (mowies) [inline .github/ISSUE_TEMPLATE/feature-request.md:10]:
```suggestion
**This is a Content Request**
```

**Comment by mowies** [comment]
@bhavyastar left some minor comments.
The DCO check is failing, please sign off your commits before pushing :)
You can check how to do it here: https://github.com/keptn/keptn.github.io/pull/1610/checks?check_run_id=11782947213

**Comment by agardnerIT** [comment]
This looks good apart from the DCO moaning.

@bhavyastar can you rebase the branch per the instructions above to keep DCO bot happy?

```
git rebase HEAD~3 --signoff
git push --force-with-lease origin issue#1593
```

This has caught me out a number of times. I now sign every commit as a habit: `git commit -sm "a message"`

Alternatively, whenever you initialise a new repository with `git init`, take a look at `.git/hooks/commit-msg.sample` which will automatically sign every commit for you.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (bhavyastar) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Okay

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+name: Content Request
```
Comment (mowies) [inline .github/ISSUE_TEMPLATE/feature-request.md:2]: pls also change the filenames according to the new issue types

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+name: Improvement Needed
```
Comment (mowies) [inline .github/ISSUE_TEMPLATE/bug-report.md:2]: pls also change the filenames according to the new issue types

**Comment by mowies** [comment]
Is this PR still active @bhavyastar ?

**Comment by bhavyastar** [comment]
Yes, it is active as well as completed but I don't why this DOC is again showing an error. Today, I will again fetch it and again make PR.

**Comment by mowies** [comment]
I still have some open comments though :)

**Comment by StackScribe** [comment]
Keptn V1 has reached EOL on December 22nd, 2023, so this PR has been closed.

**Pull request closed without merging**